README.md

```diff
@@ -66,7 +73,7 @@ sudo apt-get update
 sudo apt-get install -y kubectl
 
 # chromium
-sudo apt-get install chromium-browser
+sudo apt-get install -y chromium-browser
```

Comment:
Actually we need to delete this, `chromium` on ubuntu doesn't work well (Drag and Drop issues) so we're using firefox (which is preinstalled)

Reply:
😎